knitr/curse-dims/curse-dims.Rmd

```diff
@@ -0,0 +1,770 @@
+directly, but the next section shows that the distaince from the mode
+reduces to a well-known distribution.
+
+
+## 6. Squared Distance of Normal Draws is Chi-Square
+
+Suppose $y = (y_1, \ldots, y_D) \in \mathbb{R}^D$ and that for each $d
+\in 1{:}D$,
+
+\[
+y_d \sim \mathsf{Normal}(0, 1),
+\]
+
+is drawn independently.  The distribution of the sum of the squared
+elements of $y$ is well known to have a chi-square distribution,
+
+\[
+(y_1^2 + \cdots + y_D^2) \sim \mathsf{ChiSquare}(D).
+\]
+
+Or using ($\mathrm{L}_2$) norm notation, the squared norm of a unit
+normal has a chi-square distribution,
+
+\[
+{|| y ||}^2 \sim \mathsf{ChiSquare}(D).
+\]
+
+This means we could have drawn the curves out by using the inverse
+cumulative distribution function for the chi-square distibution (see
+the exercises).
+
+
+## 7.  Maximum Likelihood and Least Squares
+
+The intimate relation between (squared Euclidean) distance and the
+normal distribution led Gauss to formulate the notion of maximum
+likelihood and show that it reduced to minimizing a sum of squares
+(i.e., "least squares"), when estimating the location parameter of a
+normal distribution.
+
+Suppose we observe $y = (y_1, \ldots, y_N) \in \mathbb{R}^N$ and we
+assume they come from a normal distribution with unit scale $\sigma =
+1$ and unknown location $\mu$.  Then the log density is
+
+\[
+\log p(y \, | \, \mu, 1) 
+\ \propto \
+\sum_{n=1}^N \log \mathsf{Normal}(y_n \, | \, \mu, 1)
+\ \propto \
+-\frac{1}{2} \sum_{n=1}^N \left( y_n - \mu \right)^2
+\ \propto \
+-\frac{1}{2} \sum_{n=1}^N \mathrm{d}(y_n, \mu).
+\]
+
+The maximum likelihood estimate for the location $\mu$ is just the
+value that maximizes the likelihood function,
+
+\[
+\mu^* 
+\ = \
+\mathrm{argmax}_{\mu} \ p(y \, | \, \mu),
+\ = \
+\mathrm{argmax}_{\mu} \ \log p(y \, | \, \mu),
+\ = \
+\mathrm{argmax}_{\mu} - \sum_{n=1}^N \mathrm{d}(y_n, \mu)^2
+\ = \
+\mathrm{argmin}_{\mu} \sum_{n=1}^N \mathrm{d}(y_n, \mu)^2.
+\]
+
+Removing the negation and converting from maximizing to minimizing
+yields the form of the least squares estimator, because
+
+\[
+\mathrm{d}(y_n, \mu)^2 = (y_n - \mu)^2.
+\]
+
+It turns out that the parameter that maximizes the maximum likelihood
+is just the average of the observations,
+
+\[
+\mu^*
+\ = \
+\frac{1}{N} \sum_{n=1}^N y_n
+\ = \
+\bar{y}.
+\]
+
+We snuck in the standard notation $\bar{y}$ for the average of the
+elements of $y$.
+
+This was what motivated Gauss to introduce the notion of maximum
+likelihood in the first place---it provides a probabilistic motivation
+for the geometric notion of averaging. 
+
+
+## Exercises
+
+1. Define an R function for the Euclidean distance between two vectors.
+What happens if you use R's matrix multiplication, such as `u %*% v`?
+
+1.  Given $y_n \in \mathbb{R}$ for $n \in 1{:}N$, the maximum
+likelihood estimate of $\mu$ for the model 
+\[
+p(y | \mu) = \prod_{n=1}^N \mathsf{Normal}(y_n \, | \, \mu, 1)
+\]
+
```

Comment:
If that was a problem, it's fixed now and renders properly, at
least for me.

> On Jun 17, 2016, at 1:06 PM, Daniel Lee <notifications@github.com> wrote:
> 
> In knitr/curse-dims/curse-dims.Rmd:
> 
> > +This was what motivated Gauss to introduce the notion of maximum
> > +likelihood in the first place---it provides a probabilistic motivation
> > +for the geometric notion of averaging. 
> > +
> > +
> > +## Exercises
> > +
> > +1. Define an R function for the Euclidean distance between two vectors.
> > +What happens if you use R's matrix multiplication, such as `u %*% v`?
> > +
> > +1.  Given $y_n \in \mathbb{R}$ for $n \in 1{:}N$, the maximum
> > +likelihood estimate of $\mu$ for the model 
> > +\[
> > +p(y | \mu) = \prod_{n=1}^N \mathsf{Normal}(y_n \, | \, \mu, 1)
> > +\]
> > +
> 
> Sorry -- should have been clearer. It wasn't the use of the number 1 repeatedly that threw me off. In the generated html file, it starts renumbering back to 1 after the second exercise. I assumed it was due to the extra whitespace signaling to markdown that the numbering was done.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>